src/modules/dataTypes.js

```diff
@@ -23,55 +23,51 @@ const DATA_SETS = 'dataSets'
 const EVENT_DATA_ITEMS = 'eventDataItems'
 const PROGRAM_INDICATORS = 'programIndicators'
 
-const TOTALS = 'totals'
-const DETAIL = 'detail'
+export const TOTALS = 'totals'
+export const DETAIL = 'detail'
 
-const programText = i18n.t('Program')
-const selectProgramText = i18n.t('Select a program')
```

Comment:
why not use `getSelectProgramText` instead of duplicating `i18n.t('Select a program')` below? (to follow the pattern used by `getProgramText`)